benchmark/benchmark.py

```diff
@@ -0,0 +1,30 @@
+import argparse
+from sys import argv
+
+from engine import *
+
+
+def _path(x):
+    return os.path.join(os.path.dirname(__file__), '../%s' % x)
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--engine_type', type=str, required=True)

```

Comment:
use choices to give user the list of available engines

Reply:
got it, I'll include that